theme/style/style.css

```diff
@@ -744,4 +744,11 @@ pre code {
         padding-left: 1.8rem;
         box-sizing: border-box;
     }
+    
+    /* blogpost hero image to fit on mobile screens*/
+    .article .secondary img {
```

Comment:
this selector seems to be slightly wrong. Also note there are 2 modes:
1) showing the /blog/ page which lists all articles
2) showing an individual article. 

The selectors are similar but not exactly the same.